### PR TITLE
Use "rabe." prefixes in userparameter configuration files.

### PR DESCRIPTION
**Pull request by paraenggu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ EOD
 ```bash
 mkdir app/${appName}/userparameters
 
-cat > app/${appName}/userparameters/${appName}.conf <<EOD
+cat > app/${appName}/userparameters/rabe.${appName}.conf <<EOD
```
Comment (hairmare) [inline README.md]: Can we have `rabe-${appName}.conf` here? The dot looks very counterintuitive to me.

File: Makefile [hunk 2]
```diff
@@ -86,7 +86,7 @@ install-app-selinux:
 install-app-config:
 	install -d $(AGENTDDIR)
 	$(foreach app,$(APPS), \
-	    install -p -m 644 app/$(app)/userparameters/$(app).conf $(AGENTDDIR) || :; \
+	    install -p -m 644 app/$(app)/userparameters/*$(app).conf $(AGENTDDIR) || :; \
```
Comment (hairmare) [inline Makefile]: Can we have `$(PN)-$(app).conf` instead of wildcarding this?

File: README.md (hunk 1 above)
Comment (paraenggu) [inline README.md]: > Can we have rabe-${appName}.conf here?

The idea was that the file name matches with the actual user parameter prefix.

File: Makefile (hunk 2 above)
Comment (paraenggu) [inline Makefile]: This was done according to our Jabber discussion, were you preferred to have wildcards instead of a fixed prefix...

**Pull request merged**